Makefile

```
@@ -49,6 +49,20 @@ serve-local-yaml:
 	@echo "You need a routable IP address or DNS in order to address resources from inside the cluster"
 	@cd ${WORKDIR}/${LOCAL_HTTP_DIR} && python3 -m http.server ${LOCAL_HTTP_SERVER_PORT}
 
+##@ Development (with cluster)
+
+.PHONY: k8s-clean
+k8s-clean: ## Cleans all resources deployed by the setup
+	@kubectl delete --all dogus --namespace=ecosystem || true
```

Found another `ecosystem` :smile: 

```suggestion
	@kubectl delete --all dogus --namespace=$(K8S_CURRENT_NAMESPACE) || true
```